syntax = "proto3";
package dbservice;




import "model/database_delivery/dbinstance.proto";

/*
UpdateDBService请求
*/
message UpdateDBServiceRequest {
message UpdateDbservice {
    /*
    运维负责人
    */
    repeated string owner = 1 ;
    /*
    名称
    */
    string name = 2 ;
    /*
    描述
    */
    string desc = 3 ;
}
        
    
    /*
    数据库服务ID
    */
    string dbServiceId = 1 ;
    /*
    更新数据库服务body
    */
    UpdateDbservice updateDbservice = 2 ;
}

/*
UpdateDBService返回
*/
message UpdateDBServiceResponse {
message Owner {
    /*
    用户ID
    */
    string instanceId = 1 ;
    /*
    邮箱
    */
    string user_email = 2 ;
    /*
    电话号码
    */
    string user_tel = 3 ;
    /*
    用户类型
    */
    string user_type = 4 ;
    /*
    名称
    */
    string nickname = 5 ;
}
        
    
    /*
    数据库实例列表
    */
    repeated database_delivery.DBInstance dbInstance = 1 ;
    /*
    运维负责人列表
    */
    repeated Owner owner = 2 ;
    /*
    数据库服务ID
    */
    string instanceId = 3 ;
    /*
    名称
    */
    string name = 4 ;
    /*
    数据库类型
    */
    string dbType = 5 ;
    /*
    描述
    */
    string desc = 6 ;
    /*
    变更时间
    */
    int64 updatedTime = 7 ;
}

/*
UpdateDBServiceApi返回
*/
message UpdateDBServiceResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    dbservice.UpdateDBServiceResponse data = 4 ;
}

